README.md

```diff
@@ -24,7 +24,7 @@ for boot device and rootfs device respectively. In order to run this demo, the f
 
 1. Raspberry Pi 5 board;
 2. [Pi UART Debugger](https://www.waveshare.com/wiki/Pi_UART_Debugger), or any other serial console suitable for
-   Raspberry Pi 5;
+   Raspberry Pi 5 (we recommend using SH1.0 3PIN cable connected to UART Debug port to free up pins);
```

Comment:
Currently this is only the option. You can't AFAIK

Reply:
The link to the instruction that you have provided has all options so it is misleading

Reply:
I confirm, in current build we can't easily change debug UART.

Reply:
Updated